src/database/index.tsx

```diff
@@ -754,7 +754,7 @@ export const saveFacility = async (
             })
           ) {
             throw new Error(
-              "Failed to save. Case and death counts are cumulative, and can only increase over time.",
+              "Failed to save. Cumulative cases, recovered cases, and deaths can only increase over time.",
```

Comment:
I believe the changes in `src/infection-model/validators.tsx` should be ensuring that recovered cases/deaths are monotonically increasing along with cumulative cases? Otherwise I can change the message and/or add the validation logic elsewhere.

Reply:
oh yes I see that now, you're right ... I didn't quite follow the chain of function calls far enough. never mind!